src/ucp/core/ucp_ep.c

```diff
@@ -986,10 +986,11 @@ void ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config)
     config->tag.rndv.rkey_size          = ucp_rkey_packed_size(context,
                                                                config->key.rma_bw_md_map);
     config->stream.proto                = &ucp_stream_am_proto;
+    config->am_u.proto                  = &ucp_am_proto;
     config->tag.offload.max_eager_short = -1;
     max_rndv_thresh                     = SIZE_MAX;
     max_am_rndv_thresh                  = SIZE_MAX;
-
+    
```

Comment:
you may want to figure out why this whitespace change is happening - reviewers sometimes take umbrage with this kind of thing.